### PR TITLE
fix loading of fonts

### PR DESCRIPTION
**Pull request by sthuck**
Hey, 

There is an issue when loading font-awesome, because they add[ a query string to the end of their font path](https://raw.githubusercontent.com/FortAwesome/Font-Awesome/master/css/font-awesome.css), and url-loader doesn't handle it. So this regex should accept those paths with a query string. 

Not 100% sure if this pull request is necessary or is there another solution\workaround (beside putting font-awesome as a static asset and editing it manually) but this is the best solution I found...

### CODE REVIEW

**Comment by roymiloh** [comment]
Thanks!

1. Please add tests (smth like [this](https://github.com/wix/wix-node-build/blob/master/test/loaders.spec.js#L235)).
2. `?:` is not really necessary. Moreover - why not applying it (query string) on all extensions? I could imagine cases where it's relevant for images as well.

**Comment by sthuck** [comment]
@roymiloh 1+2 - done

File: test/loaders.spec.js [hunk 1]
```diff
@@ -253,6 +253,26 @@ describe('Loaders', () => {
...
+            require('./image.svg?version=1.0.2&some-other-param=value');`,
+          'src/font.ttf': _.repeat('a', 10001),
+          'src/font.woff2': _.repeat('a', 10),
+          'src/image.svg': _.repeat('a', 10001)
+        })
+        .execute('build');
+
+      const content = test.content('dist/statics/app.bundle.js');
+
+      expect(content).to.match(/"font.ttf\?\w+"/);
```
Comment (roymiloh) [inline test/loaders.spec.js]: Wouldn't better test be `expect(content).to.contain(/"font.ttf?version=1.0.0"/)`?

File: test/loaders.spec.js [hunk 2]
```diff
@@ -253,6 +253,31 @@ describe('Loaders', () => {
...
+          'src/font.ttf': _.repeat('a', 10001),
+          'src/font.woff2': _.repeat('a', 10),
+          'src/image.svg': _.repeat('a', 10001)
+        })
+        .execute('build');
+
+      const content = test.content('dist/statics/app.bundle.js');
+
+      expect(content).to.contain('font.ttf?version=1.0.0');
+      expect(content).to.match(/"font.ttf\?\w+"/);
```
Comment (roymiloh) [inline test/loaders.spec.js]: Redundant, `contain(...)` is rigorous enough.

File: test/loaders.spec.js [hunk 3]
```diff
@@ -253,6 +253,31 @@ describe('Loaders', () => {
...
+
+      expect(content).to.contain('font.woff2?version=1.0.1');
+      expect(content).to.contain('data:application/octet-stream;base64,YWFhYWFhYWFhYQo=');
+
+      expect(content).to.contain('./image.svg?version=1.0.2&some-other-param=value');
+      expect(content).to.match(/"image.svg\?\w+"/);
```
Comment (roymiloh) [inline test/loaders.spec.js]: Redundant, `contain(...)` is rigorous enough.

File: test/loaders.spec.js [hunk 4]
```diff
@@ -253,6 +253,31 @@ describe('Loaders', () => {
...
+        })
+        .execute('build');
+
+      const content = test.content('dist/statics/app.bundle.js');
+
+      expect(content).to.contain('font.ttf?version=1.0.0');
+      expect(content).to.match(/"font.ttf\?\w+"/);
+
+      expect(content).to.contain('font.woff2?version=1.0.1');
+      expect(content).to.contain('data:application/octet-stream;base64,YWFhYWFhYWFhYQo=');
```
Comment (roymiloh) [inline test/loaders.spec.js]: Prbly not _that_ related, you can remove it.

**Pull request merged**